CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Versions
 
+## x.x.x
+* Fix `<AdView/>` not calling `onAdLoaded` and `onAdLoadFailedEvent`. (https://github.com/AppLovin/AppLovin-MAX-React-Native/issues/354) 
```

Comment:
I think you tested this right? Under what circumstances would this bug occur and is it a regression of 7.0.0?

Reply:
I tested it on both iOS and Android.   This occurs only on Android, and somehow, I missed it.   This is a regression of preloading in 7.0.0.

Reply:
I would update the changelog like so then
```suggestion
* Fix native UI component banners and MRECs (`<AdView/>`) not calling the `onAdLoaded` and `onAdLoadFailed` callbacks on Android. Regression from v7.0.0. (https://github.com/AppLovin/AppLovin-MAX-React-Native/issues/354) 
```